### PR TITLE
Sephirah Update

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Couple Updates to Sephirah:
1) Lets them choose between humanoid and box form, and the color of their box.
2) Their name is now automatically generated based off which department they want
3) Gives them manager access, in a minor attempt to let them fill in for a dead manager/on lowpop
4) All huds are now inherent to the job

## Why It's Good For The Game
More Sephirah changes

## Changelog
:cl:
add: Added Sephirah Box mode
tweak: minor tweaks to sephirah access
/:cl:

### CODE REVIEW

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -90,6 +90,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/preferred_ai_core_display = "Blue"
 	var/prefered_security_department = SEC_DEPT_NONE
 	var/prefered_agent_department = "None"
+	var/prefered_sephirah_department = "Control"
+	var/prefered_sephirah_bodytype = "Humanoid"
+	var/prefered_sephirah_boxcolor = "000"
```
Comment (out-of-phaze) [inline code/modules/client/preferences.dm:95]:
```suggestion
	var/prefered_sephirah_boxcolor = "555"
```
I believe this is the minimum brightness allowed by the check below.

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -275,8 +278,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<br><br>"
 
 			dat += "<b>Custom Job Preferences:</b><BR>"
-			dat += "<a href='?_src_=prefs;preference=ai_core_icon;task=input'><b>Preferred AI Core Display:</b> [preferred_ai_core_display]</a><br>"
 			dat += "<a href='?_src_=prefs;preference=agent_dept;task=input'><b>Preferred Agent Department:</b> [prefered_agent_department]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_dept;task=input'><b>Preferred Sephirah Department:</b> [prefered_sephirah_department]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_body;task=input'><b>Preferred Sephirah Body:</b> [prefered_sephirah_bodytype]</a><BR>"
+			dat += "<a href='?_src_=prefs;preference=seph_boxcolor;task=input'><b>Preferred Sephirah Box Color:</b> [prefered_sephirah_boxcolor]</a><BR>"
```
Comment (out-of-phaze) [inline code/modules/client/preferences.dm:284]:
```suggestion
			dat += "<b>Preferred Sephirah Box Color:</b> <span style='border: 1px solid #161616; background-color: #[prefered_sephirah_boxcolor];'>&nbsp;&nbsp;&nbsp;</span> <a href='?_src_=prefs;preference=seph_boxcolor;task=input'>Change</a><BR>"
```
image
This way you can actually see the color you're using.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
Did some brief testing locally, ran into one issue and one UI thing I'd change. Also see #977, it fixes the issue you mention in the PR description.

File: code/modules/client/preferences.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/modules/client/preferences.dm]: Thanks, UI really is my weak point.

**Review by dsmith328 — APPROVED** [review]

**Pull request merged**